src/once_cell.rs

```diff
@@ -594,7 +593,7 @@ impl<T> OnceCell<T> {
                     // but we do not have the ability to initialize it.
                     //
                     // We need to wait the initialization to complete.
-                    future::poll_fn(|cx| {
+                    PollFn::new(|cx| {
```

Comment:
can't we use `async { }` here? :thinking:

Reply:
I figured out a way that we can, and I use it now.